src/server.cpp

```diff
@@ -237,7 +237,7 @@ CServer::CServer ( const int          iNewMaxNumChan,
     iMaxNumChannels ( iNewMaxNumChan ),
     iCurNumChannels ( 0 ),
     Socket ( this, iPortNumber, iQosNumber, strServerBindIP, bNEnableIPv6 ),
-    Logging(),
+    //   Logging(),
```

Comment:
Please just delete, rather than commenting out - using `git` means the original is never lost.